File: conversions/csharp_unity/Phenomenon.cs
using System;
using System.Collections.Generic;
[System.Serializable]
public class Phenomenon : BaseElement
{
    // Manifest
    public string Presence;
    public string Scope;
    public int? Duration;
    public string Intensity;
    public List<string> EmpowermentsIds; = new List<string>();
    public List<string> EnvironmentsIds; = new List<string>();
    public List<string> CarriersIds; = new List<string>();
    // Actuate
    public string Effect;
    public string CatalystsId;
    public List<string> WieldersIds; = new List<string>();
    public List<string> HandlersIds; = new List<string>();
    public List<string> EnablersIds; = new List<string>();
    public List<string> TriggersIds; = new List<string>();
    public List<string> AffinityIds; = new List<string>();
}

File: languages/typescript/Phenomenon.ts
import { BaseElement } from './base_elements';

export interface Phenomenon extends BaseElement {
  // Manifest
  presence?: string | null;
  scope?: string | null;
  duration?: number | null;
  intensity?: string | null;
  empowermentsIds?: string[] | null;
  environmentsIds?: string[] | null;
  carriersIds?: string[] | null;
  // Actuate
  effect?: string | null;
  catalystsId?: string | null;
  wieldersIds?: string[] | null;
  handlersIds?: string[] | null;
  enablersIds?: string[] | null;
  triggersIds?: string[] | null;
  affinityIds?: string[] | null;
}
